python/guide/packages/gluon/trainer.md

```diff
@@ -75,16 +98,12 @@ trainer.step(batch_size)
 net.weight.data()
 ```
 
-For all implemented methods, please refer to the
-[API reference](http://beta.mxnet.io/api/gluon-related/mxnet.optimizer.html) for
-the `optimizer` module. Besides, the
-[Dive into Deep Learning](http://en.diveintodeeplearning.org/chapter_optimization/index.html)
-book explains each optimization methods from scratch.
+For reference and implementation details about each optimizer, please refer to the [guide](http://beta.mxnet.io/guide/packages/optimizer/optimizer.html) for the `optimizer` module. The [Dive into Deep Learning](http://en.diveintodeeplearning.org/chapter_optimization/index.html) book also has a chapter dedicated to optimization methods and explains various key optimizers in great detail.
```

Comment:
Would remove reference to Dive into Deep Learning book here. And put at the end. Would link to your tutorial instead.

Reply:
done in next commit.